### PR TITLE
RATEST-290 Get rid of PageRejectedException file within the automation folder

### PR DESCRIPTION
**Pull request by Ndacyayisenga-droid**
Get rid of PageRejectedException file within the automation folder
Ticket id **RATEST-290**
Ticket url https://issues.openmrs.org/browse/RATEST-290

### CODE REVIEW

**Comment by kdaud** [comment]
@Ndacyayisenga-droid update the branch with changes from master.

**Review by kdaud — APPROVED** [review]
cc: @sherrif10

**Comment by kdaud** [comment]
@sherrif10 do you have any comment before we merge this commit?

File: qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/FormSteps.java [hunk 1]
```diff
@@ -36,8 +36,7 @@
 	@Before(RunTest.HOOK.SELENIUM_FORM)
 	public void visitDashboard() {
 		initiateWithLogin();
-		manageForm = new ManageFormsPage(driver);
-		htmlFormsPage = new HtmlFormsPage(page);
```
Comment (sherrif10) [inline qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/FormSteps.java:40]: Is this change supposed to be here

File: qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/FormSteps.java [hunk 2]
```diff
@@ -58,7 +57,7 @@ public void systemLoadsManageHtmlFormsPage() {
 	@When("a user check the availability of forms")
 	public void instantiateFormPresence() throws InterruptedException {
 		if (manageHtmlFormsPage.getElementsIfExisting(By.xpath("//*[contains(text(), '" + NAME + "')]")).isEmpty()) {
-			manageHtmlFormsPage.clickOnNewHtmlForm();
+			htmlFormsPage =	manageHtmlFormsPage.clickOnNewHtmlForm();
 			htmlFormsPage.createNewFormTest(NAME, DESCRIPTION, VERSION);
```
Comment (sherrif10) [inline qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/FormSteps.java:61]: Remove changes that are not needed .make it more clear

**Comment by sherrif10** [comment]
@Ndacyayisenga-droid  some comments above to address and get this merged.

**Comment by Ndacyayisenga-droid** [comment]
@sherrif10 those changes came in when i updated my branch with the master branch
I suppose those changes are already in master

File: qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/PageRejectedException.java [hunk 3]
```diff
@@ -1,31 +0,0 @@
-/**
- * This Source Code Form is subject to the terms of the Mozilla Public License,
- * v. 2.0. If a copy of the MPL was not distributed with this file, You can
- * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
```
Comment (sherrif10) [inline qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/PageRejectedException.java:4]: Are you not adding this class somewhere

**Comment by kdaud** [comment]
@Ndacyayisenga-droid are you still working on this?

**Pull request closed without merging**